siliconcompiler/schema/schema_cfg.py

```diff
@@ -259,6 +247,18 @@ def schema_fpga(cfg):
             tool. The part name must be an exact string match to the partname
             hard coded within the FPGA EDA tool.""")
 
+    scparam(cfg, ['fpga', family, 'lutsize'],
+            sctype='str',
```

Comment:
is this a string or an int? Seems like it should be an int

Reply:
Was afraid of using int due to serialization concerns; will make int work so that intuition and implementation match